src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/IndeterminateSentence.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model
+
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.enumerations.HistoricalTusedSource
+import java.time.LocalDate
+import java.util.UUID
+
+data class IndeterminateSentence(
+  override val offence: Offence,
+  val duration: Duration,
```

Comment:
This was my worry in creating a class for IndeterminateSentence. Do they have a duration? or any of the other properties of an `AbstractSentence`?

Reply:
And `AbstractSentence` is a `CalculableSentence` Which means an `IndeterminateSentence` cannot be an `AbstractSentence`